guides/using-helpers.md

```diff
@@ -4,4 +4,166 @@ layout: guides
+
+If you are using node.js, then the following `require` statements will ensure you have the needed Dust modules available.
+
+```
+require('dustjs-linkedin');
+require('dustjs-helpers');
+```
+
+## Logic Helpers
+
+The helpers library comes with the following logic helpers: `{@eq}`, `{@ne}`, `{@gt}`, `{@lt}`, `{@gte}`, and `{@lte}`. These helpers allow you to print content if one value compared in a certain way (e.g. equal, greater than or equal to) to another value is true. For each helper, you specify the first value with the `key` attribute and the second value with the `value` attribute. Both `key` and `value` can point to a reference or a literal value. Wrap literal values in quotes and leave references un-quoted.
+
+In the following example, the first helper looks for the value of `level` in the underlying JSON data and checks if it is equal to the literal string "master". The second checks to see if the value for the `age` reference is greater than the value of the `carRentalAge` reference.
+
+<dust-demo template-name="helpers-logic">
+<dust-demo-template>
+{@eq key=level value="master"}You are no longer a Padawan. {/eq}
+{@gt key=age value=carRentalAge}Rent a Car!{/gt}
+</dust-demo-template>
+<dust-demo-json>
+{
+  "level": "master",
+  "age": 27,
+  "carRentalAge": 25
+}
+</dust-demo-json>
+</dust-demo>
+
+### Else
+
+You can use an `{:else}` clause to do something if the comparison is false.
+
+<dust-demo template-name="helpers-else">
+<dust-demo-template>
+{@eq key=level value="master"}
+  You are no longer a Padawan.
+{:else}
+  You have much to learn, young Padawan.
+{/eq}
+</dust-demo-template>
+<dust-demo-json>
+{
+  "level": "padawan"
+}
+</dust-demo-json>
+</dust-demo>
+
+### Casting
+
+If you are comparing a literal value to one that you know is not a string (e.g. a number or a boolean), make sure to specify the `type` attribute so Dust knows how to cast the literal value.
+
+<dust-demo template-name="helpers-casting">
+<dust-demo-template>
+{! Convert "18" to a number before comparing !}
+{@gte key=age value="18" type="number"}You can vote!{/gte}
+</dust-demo-template>
+<dust-demo-json>
+{
+  "age": 20
+}
+</dust-demo-json>
+</dust-demo>
+
+## Select Helper
+
+The `@select` helper can be combined with the other logic helpers to form a `switch`-like structure, allowing you to take one action based on multiple comparisons with a single key value. You move the `key` attribute to `@select` and wrap the helper around the comparisons, specifying only a `value` attribute for each one. Each condition will be tested until one is true. When a true condition is
+found, Dust executes the condition's body and skips the rest of the conditions. You can specify what to do if none of the conditions are true using the `@default` helper.
```

Comment:
Talked to Seth about it, and @any/@none aren't in the latest working release yet (doesn't work in the dust-demo on the page), so maybe we shouldn't mention it just yet.